### PR TITLE
[ADD] Referrals: reporting doc with use case

### PR DESCRIPTION
**Pull request by larm-odoo**
No reporting doc exists for Referrals- it was a short section in the main Referrals doc. Now this is a stand-alone with a specific use case.

Original [task card](https://www.odoo.com/odoo/project/3835/tasks/4180879) for this PR.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @ksc-odoo - this is ready for a peer review!

File: content/applications/hr/referrals/reporting.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+   :align: center
+   :alt: The default report in the Referrals app.
+
+Use case: hired referrals
+-------------------------
+
+One way to use the reporting feature is to assess which employees are referring the highest quality
+applicants. This is done by examining how many of their referrals go on to become employees.
+
+In this example, data will be examined to determined which employee has the highest number of hired
```
Comment (ksc-odoo) [inline content/applications/hr/referrals/reporting.rst:45]:
```suggestion
In this example, data is examined to determined which employee has the highest number of hired
```

File: content/applications/hr/referrals/reporting.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+referrals for the current year.
+
+To view this information, first click the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the
+upper-right corner. Next, remove the current filter in the search bar. Click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` in the search bar to reveal a drop-down menu. Click
+:guilabel:`Date` in the :icon:`fa-filter` :guilabel:`Filters` column, revealing a drop-down menu of
+available time periods, and click the current year (in this example, :guilabel:`2024`).
```
Comment (ksc-odoo) [inline content/applications/hr/referrals/reporting.rst:52]:
```suggestion
upper-right corner. Next, remove the current filter in the search bar. 

Click the :icon:`fa-caret-down` :guilabel:`(down arrow)` in the search bar to reveal a drop-down menu. Click
:guilabel:`Date` in the :icon:`fa-filter` :guilabel:`Filters` column, revealing a drop-down menu of available time
periods, and click the current year (in this example, :guilabel:`2024`).
```

File: content/applications/hr/referrals/reporting.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Next, click :guilabel:`Measures` :icon:`fa-caret-down` in the top-left corner, then click
+:guilabel:`Earned Points` and :guilabel:`Employee Referral Refused` to hide those metrics. Click
+anywhere on the screen to close the drop-down menu.
```
Comment (ksc-odoo) [inline content/applications/hr/referrals/reporting.rst:56]:
```suggestion
Next, click :guilabel:`Measures` :icon:`fa-caret-down`, then deselect :guilabel:`Earned Points` and
:guilabel:`Employee Referral Refused` to hide those metrics. Click anywhere on the screen to close the
drop-down menu.
```
```suggestion
Next, click :guilabel:`Measures` :icon:`fa-caret-down` in the top-left corner, then click
:guilabel:`Earned Points` and :guilabel:`Employee Referral Refused` to hide those metrics. Click
anywhere on the screen to close the drop-down menu.
```

File: content/applications/hr/referrals/reporting.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+The information displayed shows how many total applicants each employee referred, and how many of
+those applicants were hired, for the current year.
+
+In this example, it can be determined that :guilabel:`Bob Wilson` is the most successful referrer,
+with three hired referrals, and nine total applicants he referred. Additionally, :guilabel:`Mitchell
```
Comment (ksc-odoo) [inline content/applications/hr/referrals/reporting.rst:62]:
```suggestion
with three hired referrals, and nine total referred applicants. Additionally, :guilabel:`Mitchell
```

File: content/applications/hr/referrals/reporting.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+.. tip::
+   The pivot table can be inserted into a new or existing spreadsheet, if desired. To do so, click
+   the :guilabel:`Insert in Spreadsheet` button right above the chart. A pop-up window appears,
+   asking which spreadsheet to insert the pivot chart into. Select the desired spreadsheet or
+   dashboard from the presented options, or select :guilabel:`Blank Spreadsheet` to create a new
+   one. Click :guilabel:`Confirm` and the selected spreadsheet loads, with the new table in it.
+
+   The spreadsheet is stored in the *Documents* application. This application needs to be installed
+   to use the :guilabel:`Insert in Spreadsheet` option.
```
Comment (ksc-odoo) [inline content/applications/hr/referrals/reporting.rst:80]:
```suggestion
   The pivot table can be inserted into a new or existing spreadsheet, if desired.
   
   To do so, click the :guilabel:`Insert in Spreadsheet` button right above the chart. A pop-up window
   appears, asking which spreadsheet to insert the pivot chart into. Select the desired spreadsheet or
   dashboard from the presented options. 
   
   Alternatively, select :guilabel:`Blank Spreadsheet` to create a new one. Click :guilabel:`Confirm`, and the
   selected spreadsheet loads, with the new table in it.

   The spreadsheet is stored in the **Documents** application. This application **must** be installed
   to use the :guilabel:`Insert in Spreadsheet` option.
```

**Comment by larm-odoo** [comment]
Hi @StraubCreative - this is ready for a final review!

**Review by StraubCreative — APPROVED** [review]
Looks good to me @larm-odoo, I like the use case you added here!
On to @samueljlieber for TR/merge 👍

File: content/applications/hr/referrals/reporting.rst [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+In this view, it is easy to see which :guilabel:`Medium` is the most successful.
+
+.. example::
+   In this example, both :guilabel:`Email` and :guilabel:`LinkedIn` are the mediums with the most
+   referrals, but :guilabel:`Email` has the most referrals that were hired.
+
+.. image:: reporting/employee-report.png
+   :align: center
+   :alt: The default report in the Referrals app.
+
```
Comment (samueljlieber) [inline content/applications/hr/referrals/reporting.rst:39]: It seems like this image is intended to be within the example admonition?

Also, this image is pretty illegible. I recommend retaking it 
```suggestion
.. example::
   In this example, both :guilabel:`Email` and :guilabel:`LinkedIn` are the mediums with the most
   referrals, but :guilabel:`Email` has the most referrals that were hired.

   .. image:: reporting/employee-report.png
      :align: center
      :alt: The default report in the Referrals app.
```

File: content/applications/hr/referrals/reporting.rst [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+This information can be helpful to the recruitment team, so they can determine the most active
+referrers in the company, and who is the most successful in terms of hires.
+
+.. image:: reporting/employee-counts.png
+   :align: center
+   :alt: The customized report showing which employees have the most referrals and hires.
+
+.. tip::
+   The pivot table can be inserted into a new or existing spreadsheet, if desired.
```
Comment (samueljlieber) [inline content/applications/hr/referrals/reporting.rst:76]: Suggestion to link to the relevant doc :)
```suggestion
   The pivot table can be inserted into a new or existing :doc:`spreadsheet
   <../../productivity/spreadsheet/insert>`, if desired.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo, thanks for your work on this brand new Referrals reporting doc. I have a couple suggestion for you to consider, approving!
.....
@robodoo delegate=larm-odoo

File: content/applications/hr/referrals/reporting.rst (hunk 7 above)
Comment (larm-odoo) [reply content/applications/hr/referrals/reporting.rst:76]: Ooooh i LOVE linked docs- I didn't think about that!

**Comment by larm-odoo** [comment]
@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo retry

**Pull request closed without merging**